taigaProject/src/taigaApi/controller/sprint_controller.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import FastAPI, Request, APIRouter, HTTPException, Header
+from typing import Annotated
+
+from taigaApi.model.projectRequest import ProjectRequest
+
+from taigaApi.project.getProjectBySlug import get_project_by_slug
+
+router = APIRouter()
+
+@router.post("/Sprints")
+def get_sprints_and_custom_fields_for_project(projectRequest: ProjectRequest, token: Annotated[str | None, Header()] = None):
+    project_info = get_project_by_slug(projectRequest.projectslug, token)
+    if project_info is None:
+        raise HTTPException(status_code=404, detail="Project Slug Not Found.")
+    custom_attributes = []
+    for custom_attribute in project_info["userstory_custom_attributes"]:
```

Comment:
Should we move this logic to the service layer? 
Also, pls add the condition to check if the project details are there in the cache